test/core/test_module_wasm_memory.js

```diff
@@ -6,8 +6,11 @@
  * found in the LICENSE file.
  */
 
-Module['wasmMemory'] = new WebAssembly.Memory({ 'initial': 256, 'maximum': 256,
+Module['wasmMemory'] = new WebAssembly.Memory({
+  'initial': {{{ toIndexType('256') }}},
+  'maximum': {{{ toIndexType('256') }}},
 #if MEMORY64
-  'index': 'i64'
+  'index': 'i64',
+  'address': 'i64',
```

Comment:
Out of curiosity, what is the difference in the spec between these two?

Reply:
Added a comment.